app/components/panel.tsx

```diff
@@ -3,22 +3,25 @@
 import { useRouter } from "next/navigation";
 import { useOptimistic, useTransition, useState } from "react";
 import { ChevronDownIcon } from "@heroicons/react/24/outline";
+import { FunnelIcon } from "@heroicons/react/24/outline";
 
 interface ExpandedSections {
   [key: string]: boolean;
 }
 
 export default function Panel({
-  authors,
+  selectedAuthors,
   allAuthors,
 }: {
-  authors: string[];
+  selectedAuthors: string[];
   allAuthors: string[];
 }) {
   let router = useRouter();
   let [pending, startTransition] = useTransition();
-  let [optimisticAuthors, setOptimisticAuthors] = useOptimistic(authors);
+  let [optimisticAuthors, setOptimisticAuthors] =
+    useOptimistic(selectedAuthors);
   let [expandedSections, setExpandedSections] = useState<ExpandedSections>({});
+  let [isPanelVisible, setPanelVisibility] = useState(false);
```

Comment:
How does setting this to `false` work if we're on desktop? Doesn't that make it hidden on desktop by default with no way to unhide it?

Reply:
I want it to be hidden in desktop because we instead of a toggle menu it is a side menu that is always visible